### PR TITLE
Anax v1.28.5 releasenotes cli

### PR DESCRIPTION
**Pull request by anaxceron**
Zowe CLI, ZE release notes + fixed vulnerabilities

### CODE REVIEW

File: versioned_docs/version-v1.28.x/getting-started/release-notes/v1_28_5.md [hunk 1]
```diff
@@ -39,27 +18,54 @@ Zowe Version 1.28.5 contains the bug fixes that are described in the following t
 
 #### Zowe CLI (Core)
 
-#### Zowe CLI Imperative Framework
+- Updated Imperative to version `4.18.19` to fix issues with normalizing newlines on file uploads. ([#1815](https://github.com/zowe/zowe-cli/issues/1815))
+
+- Updated Imperative to version `4.18.18` to use `@zowe/secrets-for-zowe-sdk` as the default credential manager.
```
Comment (anaxceron) [inline versioned_docs/version-v1.28.x/getting-started/release-notes/v1_28_5.md:23]: @adam-wolfe does this need a PR/issue link?

File: versioned_docs/version-v1.28.x/getting-started/release-notes/v1_28_5.md [hunk 2]
```diff
@@ -39,27 +18,54 @@ Zowe Version 1.28.5 contains the bug fixes that are described in the following t
...
 
-#### CICS Plug-in for Zowe CLI
+- Updated Imperative to include bugfixes in version `4.18.17`.
```
Comment (anaxceron) [inline versioned_docs/version-v1.28.x/getting-started/release-notes/v1_28_5.md:25]: @adam-wolfe does this need a PR/issue link?

**Comment by anaxceron** [comment]
@adam-wolfe -- can you please review the release notes for accuracy/missing information? Also, could you provide PR/issue links where necessary/helpful to the reader?

File: versioned_docs/version-v1.28.x/getting-started/release-notes/v1_28_5.md (hunk 1 above)
Comment (adam-wolfe) [reply versioned_docs/version-v1.28.x/getting-started/release-notes/v1_28_5.md:23]: Not sure. Also, we updated imperative to 4.18.17 then to 4.18.18 and 4.18.19. Does it make sense to list all of those updates if the end result is that Zowe CLI for the 1.28.5 release is using v4.18.19 of imperative? Might be a good thing to discuss with the squad.

File: versioned_docs/version-v1.28.x/getting-started/release-notes/v1_28_5.md [hunk 3]
```diff
@@ -39,27 +18,54 @@ Zowe Version 1.28.5 contains the bug fixes that are described in the following t
...
 
 ### Zowe Explorer
 
+- Replaced `keytar` dependency with `keyring` module from [`@zowe/secrets-for-zowe-sdk`](https://github.com/zowe/zowe-cli/tree/master/packages/secrets). ([#2358](https://github.com/zowe/vscode-extension-for-zowe/issues/2358))
+
 ### Vulnerabilities fixed
 
 Zowe discloses fixed vulnerabilities in a timely manner giving you sufficient time to plan your upgrades. Zowe does not disclose the vulnerabilities fixed in the latest release as we respect the need for at least 45 days to decide when and how you upgrade Zowe. When a new release is published, Zowe publishes the vulnerabilities fixed in the previous release. For more information about the Zowe security policy, see the [Security page](https://www.zowe.org/security.html) on the Zowe website.
 
 The following security issues were fixed by the Zowe security group in version 1.27.x.
```
Comment (adam-wolfe) [inline versioned_docs/version-v1.28.x/getting-started/release-notes/v1_28_5.md:41]: This should probably be updated to say "... version 1.28.5.

File: versioned_docs/version-v1.28.x/getting-started/release-notes/v1_28_5.md (hunk 1 above)
Comment (anaxceron) [reply versioned_docs/version-v1.28.x/getting-started/release-notes/v1_28_5.md:23]: Done -- thank you!

File: versioned_docs/version-v1.28.x/getting-started/release-notes/v1_28_5.md (hunk 2 above)
Comment (anaxceron) [reply versioned_docs/version-v1.28.x/getting-started/release-notes/v1_28_5.md:25]: No longer an issue

File: versioned_docs/version-v1.28.x/getting-started/release-notes/v1_28_5.md (hunk 3 above)
Comment (anaxceron) [reply versioned_docs/version-v1.28.x/getting-started/release-notes/v1_28_5.md:41]: This list includes fixes from the _last_ version. I changed this to 1.28.4.

**Review by adam-wolfe — APPROVED** [review]

**Pull request merged**